src/java.base/share/classes/sun/net/util/IPAddressUtil.java

```diff
@@ -134,22 +134,39 @@ public static byte[] textToNumericFormatV4(String src)
      * If string can't be parsed by following IETF IPv4 address string literals
      * formatting style rules (default one), but can be parsed by following BSD formatting
      * style rules, the IPv4 address string content is treated as ambiguous and
-     * {@code IllegalArgumentException} is thrown.
+     * either {@code IllegalArgumentException} is thrown, or {@code "null"} is returned.
      *
      * @param src input string
+     * @param throwIAE {@code "true"} - throw {@code IllegalArgumentException} when cannot be parsed as IPv4 address string;
+     *                 {@code "false"} - throw {@code "IllegalArgumentException"} only when IPv4 address string is ambiguous.
```

Comment:
Are these double quotes around `true`, `false`, `null` and `IllegalArgumentException` intentional? It seems odd to have those double quotes when (rightly) using `{@code}`.

Reply:
Nope, there are not intentional - a leftover from the fix prototyping stage. Will remove them.